### ISSUE TITLE
[a11y]: Pagination - Select options announced twice

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#14682]
[Original report](https://ibm-studios.slack.com/archives/C054WK4ES5S/p1694709381258889)

> All the drop downs in Pagination component
> 
> Dropdown options announced twice when we test using Jaws SR
> 
> When we navigate using UP and Down Arrow keys the options are read twice by Jaws in Firefox. NOTE The same issue found when the dropdown receives the focus using TAB and Shift TAB navigation.

**Comment by tay1orjones** [comment]
I think this is due to a change made recently to the `Select` where the `title` attribute is updated on selection. This causes the selected item to be read twice "Selected 1, 1"

**Comment by guidari** [comment]
I just tested the issue in the JAWS and it is not annoucing twice anymore
